onecodex/analyses.py

```diff
@@ -49,7 +50,7 @@ def _guess_normalized(self):
         """
         return (
             getattr(self, "_normalized", False)
-            or getattr(self, "_field", None) == "abundance"
+            or AbundanceMetric.has_value(self._metric)
```

Comment:
so `self._metric` cannot be None/undefined anymore?

Reply:
No, it shouldn't be.